docs/vue/your-first-app.md

```diff
@@ -89,45 +88,61 @@ npm install @capacitor/camera @capacitor/preferences @capacitor/filesystem
 
 ### PWA Elements
 
-Some Capacitor plugins, including the Camera API, provide the web-based functionality and UI via the Ionic [PWA Elements library](https://github.com/ionic-team/pwa-elements).
+Some Capacitor plugins, including the [Camera API](../native/camera.md), provide the web-based functionality and UI via the Ionic [PWA Elements library](https://github.com/ionic-team/pwa-elements).
 
 It's a separate dependency, so install it next:
 
 ```shell
 npm install @ionic/pwa-elements
 ```
 
-After installation, open up the project in your code editor of choice.
-
 Next, import `@ionic/pwa-elements` by editing `src/main.ts`.
 
-```tsx
-// Above the createApp() line
+```ts
+import { createApp } from 'vue';
+import App from './App.vue';
+import router from './router';
+
+import { IonicVue } from '@ionic/vue';
+// CHANGE: Add the following import
 import { defineCustomElements } from '@ionic/pwa-elements/loader';
+
+/* Ionic styles are not shown in this example to keep it brief but will be included in the Ionic package downloaded for your app. Do not remove them. */
```

Comment:
This does not follow the same flow as the Angular example. Should it be like the following:

```suggestion
/* ...existing Ionic styles... */
```

Or should we just include them?

Reply:
https://github.com/ionic-team/ionic-docs/pull/4197/commits/233761afa2b2790f12aea40055a4ff48a306921b and https://github.com/ionic-team/ionic-docs/pull/4197/commits/cef88ca09302d250291f4186dde515d7d2b450ef